addons/stock/views/stock_inventory_views.xml

```diff
@@ -136,7 +136,9 @@
                             </div>
                             <div class="o_kanban_record_bottom">
                                 <div class="oe_kanban_bottom_left">
-                                    <i class="fa fa-clock-o" role="img" aria-label="Date" title="Date"/> <t t-esc="record.date.value"/>
+                                </div>
+                                <div class="oe_kanban_bottom_right">
+                                    <t t-esc="record.date.value and record.date.value.split(' ')[0] or False"/>
```

Comment:
is it the best way to dispay a datetime in date on a kanban? are they no widgets?

Reply:
@sle-odoo Done